components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -339,6 +339,14 @@ interface PageHandlerFactory {
                         brave_wallet_auto_pin_service);
 };
 
+// lifttime option in sync with CreatePermissionLifetimeOptions()
+enum PermissionLifetimeOption {
+  kPageClosed = 0,
+  k24Hours = 1,
+  k7Days = 2,
+  kForever = 3
+};
```

Comment:
please add a unit test to match this with `CreatePermissionLifetimeOptions()` result (check count and `EXPECT_EQ(option.lifetime)` for each option).

Reply:
That is indeed a good suggestion, fixed in https://github.com/brave/brave-core/commit/aa7c8e2de55cdf3d10bd4520253c8422046fb29e